Hyperlint Automation: Meta Description Fixes (#19654)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx

---
pcx_content_type: concept
title: WARP architecture

> description: Explore how Cloudflare WARP client routes DNS and IP traffic through Zero Trust policies, enhancing security across Windows, macOS, Linux, iOS, and Android.

Reviewer: ```suggestion
description: Explore how the Cloudflare WARP client routes DNS and IP traffic to apply your Zero Trust policies.
```
